README.rst

```diff
@@ -62,14 +62,16 @@ Compatibility
 
 llvmlite works with Python 3.6 and greater.
 
-As of version 0.33.0, llvmlite requires LLVM 9.0.x.
+As of version 0.34.0, llvmlite requires LLVM 10.0.x. on all architectures
+except ``aarch64`` which requires LLVM 9.0.x.
 
 Historical compatibility table:
 
 =================  ========================
 llvmlite versions  compatible LLVM versions
 =================  ========================
-0.33.0 - ...       9.0.x
+0.34.0 - ...       10.0.x (9.0.x for  ``aarch64`` only)
```

Comment:
"for  ``aarch64`` only" will depend on whether we can get `armv7l` to build

Reply:
this is good, since we can get `armv7l` to build and test.